app/src/main/res/values/strings.xml

```diff
@@ -289,10 +281,20 @@
   <string name="my_downloads">My Downloads</string>
   <string name="tab_downloads">Downloads</string>
   <string name="tab_updates">Updates (2)</string>
-  <!-- HomeActivityBackDialog -->
+  <!-- HomeActivity -->
+  <string name="home_activity_title">Home</string>
   <string name="home_activity_back_dialog_message">Would you like to exit your profile?</string>
   <string name="home_activity_back_dialog_cancel">Cancel</string>
   <string name="home_activity_back_dialog_exit">Exit</string>
+  <string name="nav_header_desc">Navigation header</string>
+  <string name="menu_home">Home</string>
```

Comment:
Remove this and other `Home` related strings and keep only one string `home_activity_title` and use it everywhere.

Reply:
Can you please elaborate more. Will I have to remove all the strings which I have put inside HomeActivity and if yes then what else I need to do ?

Reply:
No, you just need to remove all those strings whose value is equal to `Home`. For example you can keep `home_activity_title` and remove `menu_home` and wherever `menu_home` was getting used earlier you can use `home_activity_title` instead. 
Similarly if you find any other string whose value is `Home` do the same steps for that too.

@vrajdesai78